[33mWARNING: Non existent mount point (file) in container: '/var/singularity/mnt/final/usr/bin/nvidia-debugdump'
[0m[33mWARNING: Non existent mount point (file) in container: '/var/singularity/mnt/final/usr/bin/nvidia-persistenced'
[0m[33mWARNING: Non existent mount point (file) in container: '/var/singularity/mnt/final/usr/bin/nvidia-cuda-mps-control'
[0m[33mWARNING: Non existent mount point (file) in container: '/var/singularity/mnt/final/usr/bin/nvidia-cuda-mps-server'
[0m/home/ucapnje
/home/ucapnje/share/DeepMass/run_scripts
nuber of expected files = 377500.0
loading mask 

(256, 256)
loading noisy/input data
loaded first file: /home/ucapnje/share/DeepMass/run_scripts/../picola_training/nicaea_rescaled_nongauss_noise/training_dataB00/sv_training_wiener.npy
Using TensorFlow backend.
Traceback (most recent call last):
  File "make_single_training_files.py", line 47, in <module>
    train_array_noisy = script_functions.load_data_preallocate(list(noisy_files[:]), n_images_per_file)
  File "/home/ucapnje/share/DeepMass/run_scripts/script_functions.py", line 93, in load_data_preallocate
    data_array[:n_images_per_file[0]] = first_file
ValueError: could not broadcast input array from shape (6500,256,256,1) into shape (5000,256,256,1)
